lib/beaker-puppet/install_utils/foss_utils.rb

```diff
@@ -1414,6 +1414,7 @@ def install_packages_on(hosts, package_hash, options = {})
             :solaris_11    => /solaris-11/,
             :windows       => /windows/,
             :eos           => /^eos-/,
+            :sles          => /sles/,
```

Comment:
not sure if we still need this since I fixed up how defaults are loaded with https://github.com/puppetlabs/beaker/pull/1532

Reply:
Yes: https://github.com/puppetlabs/beaker-puppet/blame/24cf1cdac0a1bee2f9ab7b7c2cb1fdb7c11b5c8d/lib/beaker-puppet/install_utils/foss_utils.rb#L1417